src/main/java/catan/services/impl/PlayServiceImpl.java

```diff
@@ -36,21 +41,24 @@
     private Logger log = LoggerFactory.getLogger(PlayService.class);
 
     public static final String ERROR_CODE_ERROR = "ERROR";
+    public static final String CARDS_ARE_OVER_ERROR = "CARDS_ARE_OVER";
```

Comment:
introducing new error code was agreed with front-end developer in common chat. he said: introduce new error code like "ZAKONCHILIS" and change story description.